packages/aws-cdk/test/integ/cli/cli.integtest.ts

```diff
@@ -143,7 +151,17 @@ integTest('deploy', withDefaultFixture(async (fixture) => {
 integTest('deploy all', withDefaultFixture(async (fixture) => {
   const arns = await fixture.cdkDeploy('test-*', { captureStderr: false });
 
-  // verify that we only deployed a single stack (there's a single ARN in the output)
+  // verify that we only deployed both stacks (there are 2 ARNs in the output)
+  expect(arns.split('\n').length).toEqual(2);
+}));
+
+integTest('deploy all concurrently', withDefaultFixture(async (fixture) => {
+  const arns = await fixture.cdkDeploy('test-*', {
```

Comment:
I'm not sure this is safe, many stacks in the integ app were designed to be deployed in a "just-so" way. Please pick a well-defined subset of like 5 stacks or so.

Reply:
I was mainly basing this off the pre-existing `deploy-all` test case - https://github.com/aws/aws-cdk/blob/9f57e255ebad12bd3106dd567581fa335477e7f2/packages/aws-cdk/test/integ/cli/cli.integtest.ts#L151

Happy to pick a different set if you'd prefer